product_weight/tests/test_compute_bom_weight.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (C) 2015 Akretion (<http://www.akretion.com>).
+# Copyright 2021 Tecnativa - Carlos Roca
```

Comment:
@CarlosRoca13 I think that this is a minor change so you can not add yourself as a file creator..